frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -445,6 +445,12 @@ private void performActionAfterChecks(MouseEvent e, Point point) {
         actionManager.getAction(ACTION_CHECK_OUT).actionPerformed(actionEvent);
       }
     }
+
+    @UIEffect
+    private boolean isMacCtrlClick(MouseEvent e) {
+      return System.getProperty("os.name").contains("Mac OS X") &&
```

Comment:
Note that https://mkyong.com/java/how-to-detect-os-in-java-systemgetpropertyosname/ gives an even broader condition. In cases like that it's better to be more **lax** than strict... dunno how the format might change b/w different JVMs/OS versions 🤔

Reply:
In fact... is the check for OS X even needed in the first place? I think we can also show the menu if someone left-clicks with Ctrl down on other systems (which is extremely unlikely to happen by accident anyway) 😅

Reply:
hmmm i do not think jetbrains are doing such an "if-ology" about right click - at least not at that level. see

```
    PopupHandler.installPopupMenu(myGraphTable, VcsLogActionIds.POPUP_ACTION_GROUP, ActionPlaces.VCS_LOG_TABLE_PLACE);
```

from `com/intellij/vcs/log/ui/frame/MainFrame.java`

Reply:
Hmmm how is that piece related to right click or Mac tho? 🤔 

![image](https://user-images.githubusercontent.com/3383210/193094240-100ac376-6172-41da-8508-339d8fe01c70.png)

Reply:
`installPopupMenu` -> idk how it works exactly but it works for git log. it makes context menu for git log available (with both "right clicks")

Reply:
> In fact... is the check for OS X even needed in the first place? I think we can also show the menu if someone left-clicks with Ctrl down on other systems (which is extremely unlikely to happen by accident anyway)

Sounds like a good idea to me :)